gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -127,6 +127,25 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             default=None,
             help="tmp directory for dask workers, uses dask's default (/tmp) if not set",
         )
+        parser.add_argument(
+            "--run_cluster",
+            action="store_true",
+            default=False,
+            help="condition whether to run gtsfm on a cluster"
+        )
+        parser.add_argument(
+            "--cluster_workers",
+            type=str,
+            default=None,
+            nargs="+",
+            help="list of worker ip addresses for the cluster"
+        )
+        parser.add_argument(
+            "--dashboard_address",
```

Comment:
do you mean --dashboard_port?

Reply:
wanted to keep consistency with dask's dashboard_address attribute, but I agree that dashboard_port is more user-friendly, addresses in the recent commit